autodispose/src/main/java/com/uber/autodispose/observers/AutoDisposingObserver.java

```diff
@@ -23,4 +23,10 @@
  * A {@link Disposable} {@link Observer} that can automatically dispose itself.
  * Interface here for type safety but enforcement is left to the implementation.
  */
-public interface AutoDisposingObserver<T> extends Observer<T>, Disposable {}
+public interface AutoDisposingObserver<T> extends Observer<T>, Disposable {
+
+    /**
+     * @return {@link Observer} The delegate Observer that is used under the hood for introspection purposes.
```

Comment:
Nit: Did you mean to link the `Observer` here rather than inline?

I'd expect this to be like `@return The delegate {@link Observer} that is used under the hood for introspection purposes.`